defi/src/utils/normalizeChain.ts

```diff
@@ -3077,9 +3091,9 @@ export const chainCoingeckoIds = {
     geckoId: null,
     symbol: null,
     cmcId: null,
-    categories: ["EVM", "Arbitrum Orbit"],
+    categories: ["EVM", "Superchain"],
     parent: {
-      chain: "Arbitrum",
+      chain: "Optimism",
```

Comment:
![image](https://github.com/DefiLlama/defillama-server/assets/80541789/85c9e97b-eadd-4e98-a465-e31089eed89b)
given that ham bridge only supports withdrawing to base, is the parent base or optimism?

Reply:
ah yes correct my bad. I have Base as the settlement layer [in my tracker too](https://github.com/ethereum-optimism/op-analytics/blob/main/op_chains_tracking/inputs/chain_metadata_raw.csv), I was just moving too fast.

Fixing!